package.json

```diff
@@ -6,12 +6,14 @@
     "babel-preset-env": "^1.5.2",
     "chalk": "^1.1.3",
     "glob": "^7.1.2",
+    "jest": "^20.0.4",
     "lerna": "^2.0.0-rc.5",
     "mkdirp": "^0.5.1",
     "rimraf": "^2.6.1"
   },
   "scripts": {
     "build-clean": "rimraf ./packages/*/build ./packages/*/build-browser ./packages/*/build-module",
-    "build": "node ./scripts/build.js"
+    "build": "node ./scripts/build.js",
+    "test": "jest"
```

Comment:
Hmmm, not sure _watch_ should be included in the default `npm test` task to be honest 🤔 

At the moment I'm refreshing the `eslint` branch for #7 and swapping a couple of things around, and this is what I've got at the moment:

```json
  "scripts": {
    "build-clean": "rimraf ./packages/*/build ./packages/*/build-browser ./packages/*/build-module",
    "build": "node ./scripts/build.js",
    "lint": "eslint . --ignore-path .gitignore",
    "pretest": "npm run lint",
    "test": "jest",
    "watch": "jest --watch"
  }
```

I doubt `pretest` would run the ESLint task if Jest is in _watch_ mode...